web/src/helpers/filter.ts

```diff
@@ -1,3 +1,4 @@
+import { PLAIN_LINK_REG } from "@/labs/marked/parser/PlainLink";
 import { getUnixTimeMillis } from "./datetime";
 import { TAG_REG, LINK_REG } from "@/labs/marked/parser";
 
```

Comment:
```suggestion
import { TAG_REG, LINK_REG, PLAIN_LINK_REG } from "@/labs/marked/parser";
```

https://github.com/usememos/memos/blob/main/web/src/labs/marked/parser/index.ts#L22

or

```suggestion
import { PLAIN_LINK_REG } from "@/labs/marked/parser/PlainLink";
import { LINK_REG} from "@/labs/marked/parser/Link";
import { TAG_REG } from "@/labs/marked/parser/Tag";
```